src/sonic-yang-models/tests/yang_model_tests/tests_config/syslog.json

```diff
@@ -0,0 +1,38 @@
+{
+    "SYSLOG_SERVER_TEST": {
+        "sonic-syslog:sonic-syslog": {
+            "sonic-syslog:SYSLOG_SERVER": {
+                "SYSLOG_SERVER_LIST": [
+                    {
+                        "server_address": "10.11.12.13"
+                    },
+                    {
+                        "server_address": "20.1.1.1"
+                    }
+                ]
+            }
+        }
+    },
+    "SYSLOG_SERVER_1_NEG_TEST": {
+        "sonic-syslog:sonic-syslog": {
+            "sonic-syslog:SYSLOG_SERVER": {
+                "SYSLOG_SERVER_LIST": [
+                    {
+                        "server_address": ""
+                    }
+                ]
+            }
+        }
+    },
+    "SYSLOG_SERVER_2_NEG_TEST": {
```

Comment:
Hi, could you add a NEG_TEST for invalid IPv4 address such as `1000.0.0.5`
I tested locally but I don't know why it doesn't block invalid IPv4 address.

Reply:
address in the latest commit.